src/scss/_header.scss

```diff
@@ -33,4 +33,18 @@
 		height: 1px;
 	}
 
+	@each $breakpoint in ('M', 'L', 'XL') {
+		.o-header__show--#{$breakpoint} {
```

Comment:
I was trying to look towards the future as the sticky header is seen as being modular - the plan is that these low level utils should help with that, as opposed to describing each variation. I feel like the header may now require more than a readme... could we extend into a wiki?